server/lib/config/pbench-server-default.cfg

```diff
@@ -110,6 +110,12 @@ webserver = %(default-host)s
 host-info-path-prefix = pbench-results-host-info.versioned/pbench-results-host-info.URL
 host_info_url = http://%(webserver)s/%(host-info-path-prefix)s
 
+# hostport logger type uses UDP-based logging.
+[logging]
+logger_type = file
+logger_host = localhost
```

Comment:
We don't want logger_host and logger_port defined at all *UNLESS* logger_type is set to hostport.